lib/spack/spack/build_environment.py

```diff
@@ -383,11 +406,13 @@ def get_rpath_deps(pkg):
 def get_rpaths(pkg):
     """Get a list of all the rpaths for a package."""
     rpaths = [pkg.prefix.lib, pkg.prefix.lib64]
+
     deps = get_rpath_deps(pkg)
-    rpaths.extend(d.prefix.lib for d in deps
-                  if os.path.isdir(d.prefix.lib))
-    rpaths.extend(d.prefix.lib64 for d in deps
-                  if os.path.isdir(d.prefix.lib64))
+    rpaths.extend(
+        d.prefix.lib for d in deps if os.path.isdir(d.prefix.lib))
```

Comment:
you don't use custom `lib` location here, it would still add `prefix/lib` or `prefix/lib64` if those exist.

Reply:
Good catch -- I think I meant to consolidate that but did not.